graylog2-web-interface/src/pages/useIndexDefaults.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+import { useQuery } from '@tanstack/react-query';
+
+import UserNotification from 'util/UserNotification';
+import fetch from 'logic/rest/FetchProvider';
+import { qualifyUrl } from 'util/URLUtils';
+import ApiRoutes from 'routing/ApiRoutes';
+import type { IndexConfig } from 'components/configurations/IndexSetsDefaultsConfig';
+
+const fetchIndexDefaults: () => Promise<IndexConfig> = () => {
+  return fetch('GET', qualifyUrl(`${ApiRoutes.ClusterConfigResource.config().url}/org.graylog2.configuration.IndexSetsDefaultConfiguration`));
+};
+
+const useIndexDefaults = () => {
+  const { data, isLoading } = useQuery(
+    ['scope-permissions'],
```

Comment:
Is this maybe leftover from a copy/paste? I'm not super familiar with `React.useQuery` but I think we want this query key to be unique so maybe something like 'index-defaults' here instead?
```suggestion
    [index-defaults'],
```

Reply:
Good catch. Definitely a copy/paste error. Fixed.